js_modules/dagster-ui/packages/ui-core/src/assets/AutoMaterializePolicyPage/PolicyEvaluationTable.tsx

```diff
@@ -63,24 +84,132 @@ export const PolicyEvaluationTable = ({evaluationRecord, definition, selectParti
           | FlattenedConditionEvaluation<UnpartitionedAssetConditionEvaluationNodeFragment>[]
           | FlattenedConditionEvaluation<SpecificPartitionAssetConditionEvaluationNodeFragment>[]
       }
-      toggleCollapsed={toggleCollapsed}
-      collapsedRecords={collapsedRecords}
+      toggleExpanded={toggleExpanded}
+      expandedRecords={expandedRecords}
     />
   );
 };
 
+const NewPolicyEvaluationTable = ({
+  flattenedRecords,
+  expandedRecords,
+  toggleExpanded,
+}: {
+  expandedRecords: Set<string>;
+  toggleExpanded: (id: string) => void;
+  flattenedRecords: FlattenedConditionEvaluation<NewEvaluationNodeFragment>[];
+}) => {
+  const [hoveredKey, setHoveredKey] = useState<number | null>(null);
+  const isPartitioned = !!flattenedRecords[0]?.evaluation.isPartitioned;
+  return (
+    <VeryCompactTable>
+      <thead>
+        <tr>
+          <th>Condition</th>
+          <th>Result</th>
+          {isPartitioned ? <th>Partitions evaluated</th> : null}
+          <th>Duration</th>
+          {/* <th>Details</th> */}
```

Comment:
@OwenKephart @braunjj This gets removed in the new design, right?

Reply:
that's right yep